src/components/Avatar/Avatar.tsx

```diff
@@ -5,6 +5,8 @@ import { AvatarIcon, AvatarIconProps } from './AvatarIcon';
 export interface AvatarProps extends AvatarIconProps {
 	name?: string;
 	title?: string;
+	subtitle?: string;
+	hideText?: boolean;
```

Comment:
Waarom deze optie? Kun je niet gewoon geen `name` & `title` meegeven? Of is er een concrete usecase?